src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp

```diff
@@ -192,9 +192,9 @@ void medAbstractDatabaseImporter::importFile ( void )
     bool atLeastOneImportSucceeded = false;
     bool atLeastOneImportError = false;
 
-    foreach ( QString file, fileList )
+    for( QString file: fileList )
     {
-        if ( d->isCancelled ) // check if user cancelled the process
+        if ( d->isCancelled ) // check if user canceled the process
```

Comment:
both seams to be ok: https://grammarist.com/spelling/cancel/

Reply:
ok it's just my orthographic corrector to suggest this